test/unit/browser/System.js

```diff
@@ -83,7 +83,7 @@ suite('System.js', function() {
 
   test('System.set', function() {
     var store = $traceurRuntime.ModuleStore;
-    var polyfills = store.getForTesting('src/runtime/polyfills/polyfills.js');
+    var polyfills = store.get(System.version + '/src/runtime/polyfills/polyfills.js');
```

Comment:
The purpose of that test was to verify `../bin/traceur-runtime.js`in the
browser. But since we did not run it, it does nothing.  Removing is fine by
me.

On Tue, Apr 7, 2015 at 6:30 PM, Erik Arvidsson <notifications@github.com>
wrote:

> In test/unit/browser/System.js
> <https://github.com/google/traceur-compiler/pull/1871#discussion_r27936628>
> :
>
> > @@ -83,7 +83,7 @@ suite('System.js', function() {
> >
> >    test('System.set', function() {
> >      var store = $traceurRuntime.ModuleStore;
> > -    var polyfills = store.getForTesting('src/runtime/polyfills/polyfills.js');
> > +    var polyfills = store.get(System.version + '/src/runtime/polyfills/polyfills.js');
>
>
> https://github.com/google/traceur-compiler/blob/master/test/runtime.html#L13
>
> It just uses a script[src] to an es6 file. This probably broke a while
> ago. We should probably remover this html file and only use 1 html test
> runner.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/traceur-compiler/pull/1871/files#r27936628>.
>